src/components/Navbar.js

```diff
@@ -20,8 +20,12 @@ function Navbar(props) {
       <AppBar position="static" className={classes.navbar}>
           <Container>
              <Toolbar>
-                <Typography variant="h6">News</Typography>
-                <Button color="inherit">Login</Button>
+                <Typography style={{marginRight:'auto' }} variant="h6">The Funky Monkey</Typography>
```

Comment:
instead of doing ```style={{ marginRight: 'auto' }}``` try doing ```className={classes.title}``` and then in ```const styles```, add another style called title. For example... ```title: { marginRight: 'auto' }```